tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2397,9 +2407,47 @@ Status ConvertExpandDims(OpConverterParams* params) {
   // ExpandDim's ability to add an axis at end of the shape.
   int trt_axis;
   TF_RETURN_IF_ERROR(ConvertAxis(axis[0], dims.nbDims + 1, node_def.name(),
-                                 /*use_implicit_batch=*/true, &trt_axis));
+                                 params->use_implicit_batch, &trt_axis));
   if (params->validation_only) return Status::OK();
 
+#if IS_TRT_VERSION_GE(6, 0, 0, 0)
```

Comment:
Do we really need to protect this code block with #if IS_TRT_VERSION_GE? If the code inside the block can be compiled with the older versions of TRT, I think we should remove #if here because we should have a common logic elsewhere to make sure we won't see dynamic dimensions here for the older versions of TRT.

Reply:
The block that needs to be protected is moved into SliceShape. The ifdef is needed there because the addShape() method of INetworkDefinition requires TRT 6.